01-js-intro-and-jquery/css/style.css

```diff
@@ -0,0 +1,24 @@
+.navigation, .content, .sidebar{
+    display: inline-block;
+}
+
+section {
+    text-align: center;
+}
+
+h1 {
+    text-align: center;
+}
+
+html {
+    background-color: silver;
+}
+
+.fade-in-paragraph {
+    display: none;
+}
+
+.query-buttons{
```

Comment:
**Mega-minor:** space before ``{``. Also, you could call this ``.query-button`` (singular), since it's going to always be applied to a single element.